### PR TITLE
Release notes - 2020-03

### PR DESCRIPTION
**Pull request by herflis**

### CODE REVIEW

File: _updates/2020-03-12-march.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Based on the feedbacks that we are getting continuously we've made some desing changes in both dark and light themes to make the ui cleaner and more usable. 💅
+
+ ![Admin-ui design update](/img/new-adminui-design.png "Admin-ui design update")
+
+- Upload is only seen and possible to choose from 'Add new' dropdown if there's possible to add Files (actually content types using the file handler and able to be uploaded).
+- The tree refactored, the basic material-ui Tree component is changed to VirtualizedTree that makes browsing the tree more seemless and it has also reduced the amount of downloaded data. 🎉🎉
+
+### DMS demo@2.0.0
+
+- DMS demo is now a simple example like the other ones (todo app, image gallery, etc), so it is moved to another url and also loss some of its funcionalities which were not really document management related.
```
Comment (zoltanbedi) [inline _updates/2020-03-12-march.md]: Shouldn't this be lost? I'm not sure.
```suggestion
- DMS demo is now a simple example like the other ones (todo app, image gallery, etc), so it is moved to another url and also lost some of its funcionalities which were not really document management related.
```

File: _updates/2020-03-12-march.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- The repository behind the dms demo is also changed to dev.demo.sensenet.com to unify the current example apps working with the same backend.
+- User management features (adding, removing users, group and role management) are removed from the DMS demo, along google authentication and the workspave favoriting and following features.
```
Comment (zoltanbedi) [inline _updates/2020-03-12-march.md]:
```suggestion
- User management features (adding, removing users, group and role management) are removed from the DMS demo, along google authentication and the workspace favoriting and following features.
```

File: _updates/2020-03-12-march.md (hunk 1 above)
Comment (herflis) [reply _updates/2020-03-12-march.md]: 'has lost' will be the solution :)

**Review by zoltanbedi — APPROVED** [review]

**Review by enikonemeth — APPROVED** [review]
😹

**Pull request merged**